be/functions/src/tests/verify-server-timezone.sh

```
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+###############################################################################
+# 서버 타임존 검증 스크립트
+# Usage: ./verify-server-timezone.sh
+###############################################################################
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+FUNCTIONS_DIR="$(cd "$SCRIPT_DIR/../.." && pwd)"
+
+cd "$FUNCTIONS_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)